code/style.css

```diff
@@ -1,7 +1,431 @@
+
+body {
+  margin: 0;
+  font-family: Arial, Helvetica, sans-serif;
+  position: absolute;
+}
+
+#hero {
+  width: 100vw;
+  height: 100vh;
+  overflow-x: hidden;
+  margin: 0;
+}
+
+#hero video {
+  width: 100%;
+  height: 100%;
+  object-fit: cover;
+  position: fixed;
+  top: 0;
+  left: 0;
+  z-index: 10;
+}
+
+#hero .headerText, .navbar {
+  position: relative;
+  z-index: 20;
+}
+
+.flexParentHeader {
+  display: flex;
+  flex-direction: column;
+}
+
+.headerText {
+  display: flex;
+  width: 100%;
+  margin: auto;
+  margin-top: 30px;
+  justify-content: center;
+  font-family: "permanent marker";
+  font-size: 34px;
+  color: #FFF7AE;
+  text-shadow: 2px 2px #9D98FA;
+}
+
+/* NAVBAR */
+
+.navbar {
+  display: flex;
+  flex-direction: row;
+  width: 100vw;
+  margin: auto;
+  justify-content: center;
+  align-items: center;
+  margin-bottom: 30px;
+}
+
+ul {
+  width: 100vw;
+  display: flex;
+  list-style-type: none;
+  justify-content: center;
+  padding-inline-start: 0;
+}
+
+li {
+  padding: 3%;
+  justify-content: center;
+}
+
+li a {
+  color: #FFF7AE;
+  font-size: 18px;
+  font-weight: 600;
+  text-decoration: none;
+}
+
+li a:hover {
+  text-decoration: underline;
+}
+
+h2 {
+  font-size: 16px;
+  margin-bottom: 8%;
+}
+
+
+/* MAIN CONTENT */
+
+.flexParent {
+  display: flex;
+  flex-direction: column;
+}
+
+
+/* FORMS CONTAINER */
+
+.formWrapper {
+  width: 80%;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  margin: auto;
+}
+
+#form {
```

Comment:
I'm curious: Did you have a specific reason to use an ID instead of a class here and in lines 135 - 137? I probably missed something while checking out the HTML, so maybe a comment in here would help make it clearer :)

Reply:
Not more specific than it being unique elements 😄